conf/app.ini

```diff
@@ -453,6 +455,8 @@ SCHEDULE = @every 24h
 ; Create new users, update existing user data and disable users that are not in external source anymore (default)
 ;   or only create new users if UPDATE_EXISTING is set to false
 UPDATE_EXISTING = true
+; Replace all Public SSH Keys that has been added manually with keys from LDAP (default false)
+REPLACE_PUBLIC_SSH_KEYS = false
```

Comment:
Wouldn't it be better to add option in source to disable users (that are created from that source) ability add keys manually then just removing his added keys. It seems nonintuitive for users perspective.